src/app/map/suitability-map-panel/suitability-map-panel.component.html

```diff
@@ -4,15 +4,16 @@
 
     <section class="map__control__section">
         <h2 class="map__control__title title">
-            <span class="title__text">Suitability Map</span>
+            <span class="title__text">{{ 'SUITABILITY.TITLE' | translate}}</span>
```

Comment:
Add space after the `translate` pipe like this `{{ 'SUITABILITY.TITLE' | translate }} `